app/src/test/java/org/simple/clinic/newentry/UserInputAgeValidatorTest.kt

```diff
@@ -95,4 +96,16 @@ class UserInputAgeValidatorTest {
     //then
     assertThat(validation).isEqualTo(Valid)
   }
+
+  @Test
+  fun `when user inputs age equal to 0 years, then return invalid error`() {
+    //given
+    val age = 0
```

Comment:
Please add a test for date of birth as well.

Reply:
Done